disruptive/resources/claim.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import annotations
+
+from typing import Any, Optional, List, Tuple
+
+import disruptive.outputs as dtoutputs
+import disruptive.requests as dtrequests
+import disruptive.errors as dterrors
+
+
+class Claim(dtoutputs.OutputBase):
+    """
+    Namespacing for claim methods.
+
+    """
+
+    KIT = 'KIT'
+    DEVICE = 'DEVICE'
+    CLAIM_ITEMS = [KIT, DEVICE]
+
+    def __init__(self, claim: dict) -> None:
+        """
+        Constructs the Claim object by unpacking raw claim response.
+
+        Parameters
+        ----------
+        claim : dict
+            Unmodified claim response dictionary.
+
+        """
+
+        # Inherit from OutputBase parent.
+        dtoutputs.OutputBase.__init__(self, claim)
+
+        # Unpack attributes from dictionary.
+        self.type: str = claim['type']
+        self.claimed_item: Claim.Kit | Claim.Device = self._resolve_type(claim)
+
+    @classmethod
+    def claim_info(cls, identifier: str, **kwargs: Any) -> Claim:
+        """
+        Get the claim information for either a kit or device.
+        The identifier can be found in the sensor QR code or
+        as an xid printed directly on the sensor.
```

Comment:
Could also include here that for kits it can be found both as text and as a QR code on the box itself.